Jenkinsfile

```diff
@@ -1,100 +1,54 @@
-node('docker') {
-    /* clean out the workspace just to be safe */
-    deleteDir()
+timestamps {
+    node('docker') {
+        /* clean out the workspace just to be safe */
+        deleteDir()
 
-    /* Grab our source for this build */
-    checkout scm
+        /* Grab our source for this build */
+        checkout scm
 
-    /* Make sure our directory is there, if Docker creates it, it gets owned by 'root' */
-    sh 'mkdir -p $HOME/.m2'
+        /* Make sure our directory is there, if Docker creates it, it gets owned by 'root' */
+        sh 'mkdir -p $HOME/.m2'
 
-    def uid = sh(script: 'id -u', returnStdout: true).trim()
-    def gid = sh(script: 'id -g', returnStdout: true).trim()
-
-    String containerArgs = '-v /var/run/docker.sock:/var/run/docker.sock -v $HOME/.m2:/var/maven/.m2'
-    dir('foreman-container') {
-        stage('Build/Test Foreman Container') {
-            def buildArgs = "."
-            docker.build('jenkins/foreman', buildArgs)
-            String foremanContainerArgs = '-p 3000:3000'
-            docker.image('jenkins/foreman').withRun(foremanContainerArgs) {
-                timeout(5) {
-                    waitUntil {
-                        def r = sh script: 'wget -q http://localhost:3000 -O /dev/null', returnStatus: true
-                        return (r == 0);
-                    }
-                }
-            }
-        }
-    }
```

Comment:
It seems you have removed a sanity check of the base Dockerfile. Was that intentional? Seems like it would be wise to fail early if the container will not build/run instead of waiting down the line to catch it.